libs/barista-components/secondary-nav/src/secondary-nav.scss

```diff
@@ -64,6 +93,18 @@ $dt-secondarynav-header-padding-left: 20px;
   transition: all 150ms ease;
 }
 
+.dt-secondary-nav-section-header:hover {
+  background: $gray-200;
+  text-decoration: none;
+}
+
+.dt-secondary-nav-section-header:hover .dt-secondary-nav-section-title {
+  color: $turquoise-700;
+}
+.dt-secondary-nav-section-header .dt-secondary-nav-section-title {
+  width: calc(100% - 52px);
```

Comment:
Could we quickly comment here, how the 52px come into existence. This is a very magic number.
It would be great if we could calculate this based on some sass variables (if this is calculatable). If not a comment will suffice.

Reply:
<img width="244" alt="Screen Shot 2020-03-13 at 2 37 21 PM" src="https://user-images.githubusercontent.com/6361609/76650382-e69f6080-6538-11ea-8194-7c332c1ce012.png">

The 52px = icon width + padding. I will add a comment unless you know of a sass var for the icon width?